### PR TITLE
Populate redis DB with default hashlist

### PR DESCRIPTION
**Pull request by Darshini-Parikh93**
- Adding list of default owner container, supply chain toolkit
manufacturer and supply chain toolkit reseller public key hash
to the redis database before starting container using demo
folder.

Signed-off-by: Darshini Parikh <darshini.k.parikh@intel.com>

### CODE REVIEW

File: demo/startRedis.sh [hunk 1]
```diff
@@ -0,0 +1,9 @@
+redis-server --daemonize yes && sleep 1
```
Comment (trbehera) [inline demo/startRedis.sh:1]: It would be good to put some comments here explaining what is being done.

File: demo/hashlist.redis [hunk 2]
```diff
@@ -0,0 +1,9 @@
+HSET OP_KEYS_WHITELIST "707B6451B8319C28E412F847E17BB87995441AF356007A03A3A4AC7745A5223D" 1
```
Comment (trbehera) [inline demo/hashlist.redis:1]: Is there a way to put a comment in this file explaining which hash pertains to which certificate?

File: demo/startRedis.sh (hunk 1 above)
Comment (Darshini-Parikh93) [reply demo/startRedis.sh:5]: Done

File: demo/hashlist.redis (hunk 2 above)
Comment (Darshini-Parikh93) [reply demo/hashlist.redis:2]: Done

File: demo/hashlist.redis [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+# Supply Chain Toolkit - Reseller ECDSA 256 public key hash
+HSET OP_KEYS_WHITELIST "85A481BBC2DA15EDD7301FF92BA2BB60093D5864A8207F9D78A399B32AB4CFF4" 1
+
+# Supply Chain Toolkit - Reseller ECDSA 384 public key hash
+HSET OP_KEYS_WHITELIST "31726603CB0751BFB926B6436369265557855744338FFC3307693E0D14D5241D" 1
+
+# Supply Chain Toolkit - Reseller RSA 2048 public key hash
+HSET OP_KEYS_WHITELIST "2ED65928AD50CB8542E648B9CD5C8B4BFB76DA870C723B16464F49F5140F7098" 1
+
+# Owner Container ECDSA 256 public key hash
```
Comment (trbehera) [inline demo/hashlist.redis:19]: Replace 'Owner Container' with IOT Platform SDK

File: demo/startRedis.sh [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+# putting the process to sleep state for 1 second
+redis-server --daemonize yes && sleep 1
+
+# Adding default hashlist to the redis-server
+redis-cli < /usr/local/hashlist.redis
+
+# Saving the added entries
+redis-cli save
+
+# closing the cli prompt
```
Comment (trbehera) [inline demo/startRedis.sh:13]: Can be rephrased as 'Stopping the 'redis-server' instance' ?

File: demo/startRedis.sh [hunk 5]
```diff
@@ -0,0 +1,17 @@
...
+redis-cli shutdown
```
Comment (prateekcr) [inline demo/startRedis.sh:14]: Is stopping redis-cli and restarting redis-server needed?  'Save' from above should be sufficient. Subsequently, is sleep really needed above?

File: demo/hashlist.redis (hunk 3 above)
Comment (Darshini-Parikh93) [reply demo/hashlist.redis]: Done

File: demo/startRedis.sh (hunk 4 above)
Comment (Darshini-Parikh93) [reply demo/startRedis.sh]: Updated the comment

File: demo/startRedis.sh (hunk 5 above)
Comment (Darshini-Parikh93) [reply demo/startRedis.sh:10]: yes. Only adding redis-cli commands results in connection refused. So the redis-server commands are required

**Pull request merged**